packages/breadcrumbs/src/BreadcrumbItem.ts

```diff
@@ -0,0 +1,130 @@
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    CSSResultArray,
+    html,
+    PropertyValues,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { property } from '@spectrum-web-components/base/src/decorators.js';
+import { Focusable } from '@spectrum-web-components/shared/src/focusable.js';
+import { LikeAnchor } from '@spectrum-web-components/shared/src/like-anchor.js';
+import chevronStyles from '@spectrum-web-components/icon/src/spectrum-icon-chevron.css.js';
+import '@spectrum-web-components/icons-ui/icons/sp-icon-chevron100.js';
+
+import styles from './breadcrumb-item.css.js';
+
+export interface BreadcrumbSelectDetail {
+    value: string;
+}
+
+export class BreadcrumbItem extends LikeAnchor(Focusable) {
+    public static override get styles(): CSSResultArray {
+        return [styles, chevronStyles];
+    }
+
+    @property()
+    public value: string | undefined = undefined;
+
+    @property({ attribute: 'is-menu', type: Boolean })
+    public isMenu = false;
+
+    @property({ type: Boolean })
+    public isLastOfType = false;
+
+    public override get focusElement(): HTMLElement {
+        return this.shadowRoot.querySelector('#item-link') as HTMLElement;
+    }
+
+    override connectedCallback(): void {
+        super.connectedCallback();
+
+        if (!this.hasAttribute('role')) {
+            this.setAttribute('role', 'listitem');
+        }
+    }
+
+    private announceSelected(value: string): void {
+        const selectDetail: BreadcrumbSelectDetail = {
+            value,
+        };
+
+        const selectionEvent = new CustomEvent('breadcrumb-select', {
+            bubbles: true,
+            composed: true,
+            detail: selectDetail,
+        });
+
+        this.dispatchEvent(selectionEvent);
+    }
+
+    protected handleClick(event?: Event): void {
+        if (!this.href && event) {
+            event.preventDefault();
+        }
+
+        if (!this.href || event?.defaultPrevented) {
+            if (this.value && !this.isLastOfType) {
+                this.announceSelected(this.value);
+            }
+        }
+    }
+
+    protected renderLink(): TemplateResult {
+        return html`
+            <a
+                id="item-link"
+                href=${ifDefined(!this.isLastOfType ? this.href : undefined)}
+                tabindex=${ifDefined(!this.isLastOfType ? 0 : undefined)}
```

Comment:
This is _risky_. The last item will not be read by screen reader.

Reply:
I was inspired here by the HTML in [Spectrum's CSS documentation](https://opensource.adobe.com/spectrum-css/breadcrumb.html), they also add some CSS on `[tabindex=0]` which should not be applied on the last item.
Would you suggest here to keep the `tabindex=0` for the last breadcrumb item, and add some CSS to remove the styles which are coming from Spectrum CSS?

Reply:
Yes adding a tab stop on the selected item would be very helpful for the screen readers to pick up! I think it is advisable from the APG guidelines!